Backpack/src/androidTest/java/net/skyscanner/backpack/util/BpkThemeTest.kt

```diff
@@ -33,7 +33,7 @@ class BpkThemeTest {
 
   @Test
   fun test_primary_color() {
-    Assert.assertEquals("default", blue500, BpkTheme.getPrimaryColor(activity))
+    Assert.assertEquals("default", skyBlue, BpkTheme.getPrimaryColor(activity))
```

Comment:
There's some inconsistency in checkbox style. We need to verify it and select only one option.

Reply:
done